subprojects/logging/src/main/java/org/gradle/internal/logging/console/BuildStatusRenderer.java

```diff
@@ -77,19 +79,18 @@ public void onOutput(OutputEvent event) {
                 phaseProgressed(progressEvent);
             }
         }
+
+        listener.onOutput(event);
+
+        if (event instanceof UpdateNowEvent | event instanceof EndOutputEvent) {
```

Comment:
Wow... good catch. Somehow CI is even passing with this change, and I don't really know what this _does_. Will fix.